docs/tutorial/tutorial-1.rst

```diff
@@ -22,47 +24,51 @@ Put the following code into ``helloworld\app.py``, replacing the old code:
               self.c_input.value = '???'
 
       def startup(self):
+          # Create a main window with a name matching the app
           self.main_window = toga.MainWindow(self.name)
-          self.main_window.app = self
 
-          # Tutorial 1
-          c_box = toga.Box()
+          # Create a main content box
           f_box = toga.Box()
+          c_box = toga.Box()
           box = toga.Box()
 
           self.c_input = toga.TextInput(readonly=True)
           self.f_input = toga.TextInput()
 
-          c_label = toga.Label('Celcius', alignment=toga.LEFT_ALIGNED)
-          f_label = toga.Label('Fahrenheit', alignment=toga.LEFT_ALIGNED)
-          join_label = toga.Label('is equivalent to', alignment=toga.RIGHT_ALIGNED)
+          self.c_label = toga.Label('Celcius', style=Pack(text_align=LEFT))
+          self.f_label = toga.Label('Fahrenheit', style=Pack(text_align=LEFT))
+          self.join_label = toga.Label('Is equivalent to', style=Pack(text_align=RIGHT))
 
           button = toga.Button('Calculate', on_press=self.calculate)
 
           f_box.add(self.f_input)
-          f_box.add(f_label)
+          f_box.add(self.f_label)
 
-          c_box.add(join_label)
+          c_box.add(self.join_label)
           c_box.add(self.c_input)
-          c_box.add(c_label)
+          c_box.add(self.c_label)
 
           box.add(f_box)
           box.add(c_box)
           box.add(button)
 
-          box.style.set(flex_direction='column', padding_top=10)
-          f_box.style.set(flex_direction='row', margin=5)
-          c_box.style.set(flex_direction='row', margin=5)
+          box.style.update(direction=COLUMN, padding_top=10)
+          f_box.style.update(direction=ROW, padding=5)
+          c_box.style.update(direction=ROW, padding=5)
+
+        self.c_input.style.update(flex=1)
```

Comment:
Is it just me, or is there something odd going on with the indentation here?

Reply:
Thanks @freakboy3742!